client/layout/oauth-client/index.jsx

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
Maybe consider using `OAuthClientMasterbar`? It feels strange that a Layout component would purely consist of a top navigation strip.